graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/java/DataTypeGenerator.kt

```diff
@@ -167,11 +167,60 @@ abstract class BaseDataTypeGenerator(
 ) {
     internal val typeUtils = TypeUtils(packageName, config, document)
 
+    object ParserConstants {
+        const val NAME = "name"
+        const val REASON = "reason"
+        const val CUSTOM_ANNOTATION = "annotate"
+        const val DEPRECATED = "deprecated"
+        const val REPLACE_WITH_STR = ", replace with "
+    }
```

Comment:
Do we really need this again? 

Could we use the the constants we had created earlier from a common class? Anyone modifying say kotlin or java, would have keep doing it for both separately.

Reply:
Moved to shared